### PR TITLE
feat: Add ability to change patient appointment status

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Add ability to change appointment status. If the status is `CheckeIn` then we redirect the user to the `outpatient` services. This is based on the available designs.

## Screenshots
Peek 2022-09-07 07-00

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/src/change-appointment-status/change-appointment-status.component.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+              ))}
+            </RadioButtonGroup>
+          </FormGroup>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button kind="secondary" onClick={closeModal}>
+          {t('cancel', 'Cancel')}
+        </Button>
+        <Button onClick={handleChangeAppointmentStatus}>{t('exitAndChangeStatus', 'Exit and change status')}</Button>
```
Comment (dkayiwa) [inline packages/esm-appointments-app/src/change-appointment-status/change-appointment-status.component.tsx:97]: When i first saw `Exit` i quickly looked for another button for saving but instead saw `Cancel`. Then i came back and noticed that it is `Exit` and then `change status`. I guess you now understand what am saying. 😊

Comment (dkayiwa) [reply packages/esm-appointments-app/src/change-appointment-status/change-appointment-status.component.tsx:97]: And on another note, i have seen this pattern all over 03. In the old days, such a window would simply have `Change status` or `Save status` without `Exit`. And it was the trend that the popup window would automatically close after clicking any of the Save or Cancel buttons. Exit was mostly reserved to exiting the entire application. `Close` would be for such small popups, if we had to. Could this be a sign of my old age and out of touch with the new modernity? 😊

**Comment by CynthiaKamau** [comment]
Great job Donald, i have left a comment

**Review by CynthiaKamau — APPROVED** [review]
LGTM

File: packages/esm-appointments-app/src/change-appointment-status/change-appointment-status.component.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+              ))}
+            </RadioButtonGroup>
+          </FormGroup>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button kind="secondary" onClick={closeModal}>
+          {t('cancel', 'Cancel')}
+        </Button>
+        <Button onClick={handleChangeAppointmentStatus}>{t('exitAndChangeStatus', 'change status')}</Button>
```
Comment (makombe) [inline packages/esm-appointments-app/src/change-appointment-status/change-appointment-status.component.tsx:97]:
```suggestion
        <Button onClick={handleChangeAppointmentStatus}>{t('exitAndChangeStatus', 'Change status')}</Button>
```

**Review by makombe — APPROVED** [review]
Looks good. Thanks

**Pull request merged**